### PR TITLE
Add blog

### PR DESCRIPTION
**Pull request by ericdarbra**

### CODE REVIEW

File: staff/ericdarbra/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
Comment (mikelpmc) [inline staff/ericdarbra/blog/index.html]: te falta el meta description

<meta name="description" content="Free Web tutorials">

[meta tag](https://www.w3schools.com/tags/tag_meta.asp)

File: staff/ericdarbra/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```
Comment (doboapps) [inline staff/ericdarbra/blog/index.html:4]: You need here a favicon... 😗

Comment (ericdarbra) [reply staff/ericdarbra/blog/index.html:4]: 🖕

File: staff/ericdarbra/blog/index.html (hunk 1 above)
Comment (mikelpmc) [reply staff/ericdarbra/blog/index.html]: 👍

**Review by mikelpmc — APPROVED** [review]

**Review by doboapps — APPROVED** [review]
👍 good code @ericdarbra

**Pull request merged**